universal_transfer_operator/transfers/universal_transfer_operator.py

```diff
@@ -0,0 +1,78 @@
+from __future__ import annotations
+
+from typing import Any
+
+from airflow.models import BaseOperator
+from airflow.utils.context import Context
+from transfers.constants import TransferMode
+from transfers.data_providers import cleanup, create_dataprovider
+from transfers.datasets.base import UniversalDataset as Dataset
+from transfers.integrations import get_transfer_integration
+from transfers.utils import check_if_connection_exists
+
+from astro.constants import LoadExistStrategy
+
+
+class UniversalTransferOperator(BaseOperator):
+    """
+    Transfers all the data that could be read from the source Dataset into the destination Dataset. From a DAG author
+    standpoint, all transfers would be performed through the invocation of only the Universal Transfer Operator.
+
+    :param source_dataset: Source dataset to be transferred.
+    :param destination_dataset: Destination dataset to be transferred to.
+    :param transfer_params: kwargs to be used by method involved in transfer flow.
+    :param transfer_mode: Use transfer_mode TransferMode; native, non-native or thirdparty.
+    :param if_exists: Overwrite file if exists. Default False.
+
+    :return: returns the destination dataset
+    """
+
+    def __init__(
+        self,
+        *,
+        source_dataset: Dataset,
+        destination_dataset: Dataset,
+        transfer_params: dict | None = {},
+        transfer_mode: TransferMode | None = TransferMode.NONNATIVE,
+        if_exists: LoadExistStrategy = "replace",
+        **kwargs,
+    ) -> None:
+        self.source_dataset = source_dataset
+        self.destination_dataset = destination_dataset
+        self.transfer_mode = transfer_mode or TransferMode.NONNATIVE
```

Comment:
since we have already set a default in constructor, we can remove this:

```suggestion
        self.transfer_mode = transfer_mode
```

Reply:
[Done](https://github.com/astronomer/astro-sdk/pull/1492/commits/831d627d1a86acab300968fc56d11bc7f016b4b0)